packages/dev/core/src/Gizmos/positionGizmo.ts

```diff
@@ -208,6 +209,23 @@ export class PositionGizmo extends Gizmo implements IPositionGizmo {
         return this._planarGizmoEnabled;
     }
 
+    /**
+     * posture that the gizmo will be display
+     * When set null, default value will be used (Quaternion(0, 0, 0, 1))
+     */
+    public get customRotationQuaternion(): Nullable<Quaternion> {
+        return this._customRotationQuaternion;
+    }
+
+    public set customRotationQuaternion(customRotationQuaternion: Nullable<Quaternion>) {
+        this._customRotationQuaternion = customRotationQuaternion;
+        [this.xGizmo, this.yGizmo, this.zGizmo].forEach((gizmo) => {
+            if (gizmo) {
+                gizmo.customRotationQuaternion = customRotationQuaternion;
+            }
+        }, this);

```

Comment:
nit: you can remove `, this` (you are using an arrow function, and you don't use `this` anyway!).